ghzdac/calibrate.py

```diff
@@ -346,7 +346,7 @@ def calibrateACPulse(cxn, boardname, baselineA, baselineB, use_switch=True):
     scope.select_device(scopeID)
     p = scope.packet().\
     reset().\
-    channel(SCOPECHANNEL).\
+    channel(reg.get(keys.SSCOPECHANNEL, 'w', True, 2)).\
```

Comment:
I would suggest omitting the pattern here because if you set `2` it will get flattened as `i` which will then fail if you try to load it with the pattern `w`. The way labrad treats integral values is not as flexible as python, unfortunately. But you can omit the pattern and just do `reg.get(keys.SSCOPECHANNEL, True, 2)`.